website/docs/faqs/Troubleshooting/ide-session-unknown-error.md

```diff
@@ -0,0 +1,20 @@
+---
+title: I'm receiving an 'Your IDE session experienced an unknown error and was terminated. Please contact support'.
+description: "Add a repository when seeing IDE unknown error"
+sidebar_label: 'Receiving unknown error in the IDE'
+id: IDE
```

Comment:
sugg removing since it's unnecessary. what you have as the filename is a good URL ie `https://docs.getdbt.com/faqs/Troubleshooting/ide-session-unknown-error`

(side note: as a best practice, if the `id` key is used, the value should be in all lower case (no upper case) to avoid Vercel issues.)

```suggestion
```